doc/changelog.md

```diff
@@ -4,6 +4,30 @@
 
 <h3>Improvements</h3>
 
+* Catalyst now supports return statements inside conditionals in `@qjit(autograph=True)` compiled
+  functions.
+  [(#)]()
+
+  The following is now possible:
+
+  ```py
+  @qjit(autograph=True)
+  @qml.qnode(qml.device(backend, wires=1))
+  def f(x: float):
+    qml.RY(x, wires=0)
+
+    m = measure(0)
+    if not m:
+        return 0
+
+    ...
+
```

Comment:
```suggestion
```

Reply:
Just to make it an example that copies and pastes without error

Reply:
`...` is a valid Python expression :D